tutorials/advanced/elasticSnorkel.ipynb

```diff
@@ -0,0 +1,583 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Number of hits \n",
+      "2354\n",
+      "Result 1\n",
+      "-------------------\n",
+      "sentence\n",
+      "He is married with        three children.     \n",
+      "\n",
+      "Result 2\n",
+      "-------------------\n",
+      "sentence\n",
+      "He was married twice and had five children.      \n",
+      "\n",
+      "Result 3\n",
+      "-------------------\n",
+      "sentence\n",
+      "He was later married and had two children.   \n",
+      "\n",
+      "Result 4\n",
+      "-------------------\n",
+      "sentence\n",
+      "She got married to my father, Joseph Ewherido, with whom she had eight children, all males.\n",
+      "\n",
+      "Result 5\n",
+      "-------------------\n",
+      "sentence\n",
+      "All of the women were married (or divorced!), and some had children.   \n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "query=\"married children\"\n",
+    "searchResult = eSearch.searchIndex(\"entire\",query)\n",
+    "printResults(searchResult,\"sentence\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Search contain all values in query\n",
+    "\n",
+    "Specifying a distance parameter will force the query to only return results that contain every word in the query. Explicitly stating a distance=0 will return results where the query values appear side by side."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Number of hits \n",
+      "8\n",
+      "Result 1\n",
+      "-------------------\n",
+      "sentence\n",
+      "Dressed in a smart black shirt and white trousers, the lawyer looked in good spirits despite a difficult fight ahead.     \n",
+      "\n",
+      "Result 2\n",
+      "-------------------\n",
+      "sentence\n",
+      "Amal, who was dressed in a smart black shirt and white trousers, looked in good spirits on Monday despite a difficult fight ahead.   \n",
+      "\n",
+      "Result 3\n",
+      "-------------------\n",
+      "sentence\n",
+      "Clad in a breton top, white trousers and a matching cap, Bob, 63, looked like he couldn't wait to get underway with the festivities.       \n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "query=\"white trousers\"\n",
+    "searchResult = eSearch.searchIndex(\"entire\",query,distance=0,size=3)\n",
+    "printResults(searchResult,\"sentence\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Positional Search\n",
```

Comment:
The utility of this one made less sense to me (why can't we just use phrase search for multi-word?), maybe cut or put at end?